bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/ReolinkHandler.java

```diff
@@ -80,6 +83,7 @@ public void channelRead(@Nullable ChannelHandlerContext ctx, @Nullable Object ms
                         ipCameraHandler.snapshotUri = "/cgi-bin/api.cgi?cmd=Snap&channel="
                                 + ipCameraHandler.cameraConfig.getNvrChannel() + "&rs=openHAB"
                                 + ipCameraHandler.reolinkAuth;
+                        // We will use admin so users don't blame missing channels/features on a bug.
```

Comment:
What does this mean?

Reply:
If a user gives the binding a non admin account that is restricted to a media user level of access, the binding will report that the user can not access the features, this results in the channels getting removed and the user will first resort to pointing fingers at the binding as have a bug. Better that the camera replies the user rights are not enough and the user then knows why and does not have to delete and re-add the thing to get the channels back again.

Reply:
Ok, very usefull comment for future maintanance. If you update the comment so that it is bit more clear, as you stated in this comment it would be perfect.

Reply:
Done, increased detail of comment.